app/models/user.rb

```diff
@@ -570,6 +570,17 @@ def banned_at_date?(date)
     delegate_roles.select { |role| role.metadata.status == RolesMetadataDelegateRegions.statuses[:senior_delegate] }
   end
 
+  private def groups_with_read_access
+    return "*" if can_edit_any_groups?
+    groups = groups_with_edit_access
+
+    if can_view_banned_competitors?
+      groups += UserGroup.banned_competitors.ids
+    end
```

Comment:
Why is this case not covered by `groups_with_edit_access` directly above?

Reply:
Because users like WEC members can view banned competitors, but they cannot edit banned competitors.

Reply:
Okay, this makes sense but still looks a bit strange because one might be under the impression that you're giving Banned a kind of "special" treatment. I know this is due to the fact that they are currently the only group that could potentially have read-only access, but at least in the back of your mind think about something more generic for computing permissions, please.